.gitignore

```diff
@@ -0,0 +1,3 @@
+/node_modules/*
+dropletDB_URL.txt
```

Comment:
I added this txt file to hold the address to my DO server. I don't think it's a big deal, but I can move the file out of the repo